clans/tabs.md

```diff
@@ -88,8 +90,32 @@ Boost-urile nu se acumulează, boost-ul maxim pentru orice perk fiind 25%/membru
 
 În cazul job boost-ului, procentul este din câștigul de bază a jobului (se poate aduna cu marijuana joint, world exploration) dar nu și cu procentul de la skill-ul jobului.
 
+"Money Laundering":
+    - nivel initial 0, jucătorii nu pot schimba banii marcați pe bani normali dacă clanul din care fac parte nu deblochează acest perk;
+    - sunt necesare urmatoarele:
+        - deblocare: $250.000 & 1.000 Gold
+        - nivel 2: $450.000 & 2.000 Gold
+        - nivel 3: $1.000.000 & 3.000 Gold
+        - nivel 4: $2.000.000 & 7.000 Gold
+        - nivel 5: $5.000.000 & 15.000 Gold
+    - perk-ul ofera urmatoarea rata de conversie la spalarea banilor:
+        - nivel 1: 35% (deblocat)
+        - nivel 2: 40%
+        - nivel 3: 45% 
+        - nivel 4: 50%
+        - nivel 5: 55%
+        - > Pentru $10.000 bani marcati (1 stack in invetar) primesti $3.000 dolari (35% după deblocarea perk-ului);
+    - perk-ul ofera urmatorii timpi de asteptare la spalarea banilor:
+        - nivel 1: 60 secunde la fiecare $1.000 spalati
+        - nivel 2: 55 secunde la fiecare $1.000 spalati
+        - nivel 3: 50 secunde la fiecare $1.000 spalati
+        - nivel 4: 45 secunde la fiecare $1.000 spalati
+        - nivel 5: 40 secunde la fiecare $1.000 spalati
+        
```

Comment:
**Cat costa fiecare nivel al perk-ului "Money Laundering"?**

Costul fiecarui nivel necesar al perk-ului "Money Laundering" este urmatorul: deblocare/nivel 1 - 250.000$ & 1.000 Gold; nivel 2 - 450.000$ & 2.000 Gold; nivel 3 - 1.000.000$ & 3.000 Gold; nivel 4 - 2.000.000$ & 7.000 Gold; nivel 5 - 5.000.000$ & 15.000 Gold.



**Cum pot spala banii?**

Nu poti schimba banii marcați pe bani normali dacă clanul din care faci parte nu deblochează acest perk. Pentru a putea spala banii este nevoie ca, clanul din care faci parte sa aiba deblocat perk-ul specific pentru aceasta actiune. Perk-ul ofera urmatoarea rata de conversie la spalarea banilor: nivel 1: 35% (deblocat); nivel 2: 40%; nivel 3: 45%; nivel 4: 50%; nivel 5: 55%. Spre exemplu, la nivelul 1, pentru $10.000 bani marcati (1 stack in invetar) primesti $3.000 dolari (35% după deblocarea perk-ului).

**Cat dureaza sa spal banii?**

Timpul de asteptare necesar pentru a spala banii este in functie de nivelul perk-ului deblocat: nivel 1 - 60 secunde la fiecare 1.000$ spalati; nivel 2 - 55 secunde la fiecare 1.000$ spalati; nivel 3 - 50 secunde la fiecare 1.000$ spalati; nivel 4 - 45 secunde la fiecare 1.000$ spalati; nivel 5 - 40 secunde la fiecare 1.000$ spalati.